lib/tracker_api/endpoints/comments.rb

```diff
@@ -12,9 +12,27 @@ def get(project_id, story_id, params={})
         raise Errors::UnexpectedData, 'Array of comments expected' unless data.is_a? Array
 
         data.map do |comment|
-          Resources::Comment.new({ story_id: story_id }.merge(comment))
+          Resources::Comment.new({ client: client,
+                                   project_id: project_id,
+                                   story_id: story_id }.merge(comment))
         end
       end
+
+      def create(project_id, story_id, params={})
```

Comment:
Can you move `create` and `update` into a new `Endpoints::Comment` class? See `Endpoints::Tasks` and `Endpoints::Task`. Just trying to keep the collection and singular resource behavior together and separate.